datagen/pom.xml

```diff
@@ -52,6 +52,12 @@
             <artifactId>spark-sql_${scala.binary.version}</artifactId>
             <version>${spark.test.version}</version>
         </dependency>
+        <!-- https://mvnrepository.com/artifact/com.github.scopt/scopt -->
+        <dependency>
+            <groupId>com.github.scopt</groupId>
+            <artifactId>scopt_2.12</artifactId>
+            <version>3.7.1</version>
```

Comment:
Use this as the Spark official is using this and the jar can be found at `$SPARK_HOME/examples/jars/scopt_2.12-3.7.1.jar`. Let me know if I should assemble it in our jar.

Reply:
Yes, when using it we need explicitly write `--jars $SPARK_HOME/examples/jars/scopt_2.12-3.7.1.jar ` to actually get it into CLASS_PATH.

Do you think we should assemble it?

Reply:
Just so long as we have good instructions on how to run the tool I am fine with it. This is not a tool that has to be turn key. We just need docs on how to use it properly. 

If it is just a dependency we don't need to include it in `NOTICE`. We only need that if we copied code from scopt into our code base.

If we package the dependency with our jar, then we need to include it in the NOTICE-binary